src/app.ts

```diff
@@ -4,17 +4,16 @@ import DBConnection from './db/DBConnection';
 
 const app = new API();
 const server = app.getServer(false);
-const PORT: number = +process.env.PORT || 5000;
-const SERVER_ADDRESS: string = '0.0.0.0';
+const PORT = process.env.PORT || 5000;
```

Comment:
Should we keep the `+` here? Down below it's used with the DB_PORT

Reply:
Yep, added it back